mnefun/_mnefun.py

```diff
@@ -1683,14 +1684,13 @@ def gen_forwards(p, subjects, structurals, run_indices):
         subjects_dir = get_config('SUBJECTS_DIR')
         if structurals[si] is None:  # spherical case
             # create spherical BEM
-            bem = make_sphere_model('auto', 'auto', info, verbose=False)
+            bem = make_sphere_model(info=info, r0='auto',
+                                    head_radius='auto')
```

Comment:
why this change? The `verbose=False` is nice to avoid fitting stuff. Feel free to emit a `print(...)` if you want a useful output.